internal/workflow.go

```diff
@@ -2401,6 +2401,12 @@ type nexusClient struct {
 //
 // NOTE: Experimental
 func NewNexusClient(endpoint, service string) NexusClient {
+	if endpoint == "" {
```

Comment:
Just as a sanity check the server does not allow an empty endpoint name to be registered or an empty service right?

Reply:
Yes, we verify that.